sklearn/kernel_approximation.py

```diff
@@ -389,10 +389,10 @@ class Nystroem(BaseEstimator, TransformerMixin):
         the kernel; see the documentation for sklearn.metrics.pairwise.
         Ignored by other kernels.
 
-    degree : float, default=3
+    degree : float, default=None
```

Comment:
It has not been deprecated. Using it with a callable (in which case it was already being ignored) is deprecated.